### PR TITLE
Add TabSheet doc

### PR DESCRIPTION
**Pull request by tarekoraby**

### CODE REVIEW

**Comment by tarekoraby** [comment]
Removed "language check" as it is a copy-paste from vaadin.com/docs.

File: articles/react/components/tabs/index.adoc [hunk 1]
```diff
@@ -14,6 +14,24 @@ description: Tabs are used to organize and group content into sections that the
...
+[source,tsx]
+----
+<vaadin-tabsheet>
+  <vaadin-tabs slot="tabs">
+    <vaadin-tab id="details-tab">Details</vaadin-tab>
+    <vaadin-tab id="payment-tab">Payment</vaadin-tab>
+    <vaadin-tab id="shipping-tab">Shipping</vaadin-tab>
+  </vaadin-tabs>
+
+  <div tab="setails-tab">This is the Details tab content</div>
```
Comment (cromoteca) [inline articles/react/components/tabs/index.adoc:30]:
```suggestion
  <div tab="details-tab">This is the Details tab content</div>
```

**Review by cromoteca — CHANGES_REQUESTED** [review]

**Comment by cromoteca** [comment]
Looks like a duplicate of https://github.com/vaadin/docs/pull/2434

**Comment by tarekoraby** [comment]
Close in favor of #2434.

**Pull request closed without merging**